extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -142,7 +142,7 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  unlayoutCallback() {
```

Comment:
Reviewed this part again. I was wrong - we don't need `pauseCallback` here. I don't think we need to look at viewport for `scheduleUnload` at this time - although it's certainly possible in the future.